src/components/PullRequestStatus.tsx

```diff
@@ -99,11 +116,35 @@ function getBadges(state: PullRequestState): JSX.Element[] {
   return badges;
 }
 
+function getCheckStatusBadge(checkStatus?: CheckStatus): JSX.Element[] {
+  switch (checkStatus) {
+    case "PENDING":
+      return [CHECK_STATUS_PENDING];
+      break;
+    case "SUCCESS":
+      return [CHECK_STATUS_PASSED];
+      break;
+    case "FAILURE":
+      return [CHECK_STATUS_FAILED];
+      break;
+
+    case "ERROR":
+    case "EXPECTED":
```

Comment:
Shouldn't we return `CHECK_STATUS_PENDING` for `"EXPECTED"` too?

Reply:
good q. i haven't been able to get this value yet through my non-extensive testing. the docs are also really sparse on when this happens.